partiql-parser/src/main/antlr/PartiQL.g4

```diff
@@ -334,6 +334,20 @@ windowSortSpecList
 havingClause
     : HAVING arg=exprSelect;
 
+excludeClause
+    : EXCLUDE excludeExpr (COMMA excludeExpr)*;
+
+excludeExpr
+    : symbolPrimitive excludeExprSteps+;  // Require 1 more `excludeExprSteps`. Disallow `EXCLUDE a`
```

Comment:
Can you explain why a step is required? I imagined that the items within the `EXCLUDE` clause would follow the same resolution rules as the projection items. If ambiguities are encountered, errors are thrown.

Reply:
Also worth mentioning `EXCLUDE` is currently modeled as a binding tuple operator (evaluated before the `SELECT` projection). We are trying to disallow excluding a top-level binding explicitly (e.g. disallow `SELECT ... EXCLUDE t FROM s, t`) since we don't have a clear use case for this.